environments/staging/meta.yml

```diff
@@ -8,3 +8,5 @@ users:
   - fissionlabs
   - fission-warehouse
 secrets_backend: aws-secrets
+
+shared_mount_opts: "_netdev,nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport"
```

Comment:
I think it makes more sense for this to go into `public.yml` with the other things rather than in meta

Reply:
I have updated public.yml file with shared_mount_opts & removed it from generated_variables.